draft-ietf-oauth-jwt-introspection-response.xml

```diff
@@ -214,155 +214,154 @@ token=2YotnFZFEjr1zCsicMWpAA]]></artwork>
 	  </t>
 	  
 	  <t>If required by its policy, the authorization server MUST authenticate the 
-	  caller and check its authorization to use the token introspection 
+	  caller and validate its authorization to use the token introspection
 	  endpoint.</t>
 
 	</section>
 	
     <section anchor="jwt_response" title="JWT Response">
     
-      <t>The introspection endpoint responds with a JWT, setting the
-          "Content-Type" header to "application/jwt". This JWT is a cryptographically 
-          protected representation of the token introspection response as specified in
-          <xref target="RFC7662"/>.</t>
+     <t>The introspection endpoint responds with a JWT, setting the
+     <spanx style="verb">Content-Type</spanx> HTTP header to
+     "application/token-introspection+jwt" and the JWT <spanx style="verb">typ</spanx>
+     ("type") header to "token-introspection+jwt".</t>
+
+     <t>The JWT MUST include the following top-level claims:
+     <list hangIndent="8" style="hanging">
+         <t hangText="iss">MUST be set to the issuer URL of the authorization
+         server.</t>
+         <t hangText="aud">MUST identify the resource server receiving the token
+         introspection response.</t>
```

Comment:
Can you please add the "iat"?

<t hangText="iat">The <spanx style="verb">iat</spanx> claim indicates when the
      	  introspection response was issued by the AS.</t>